### ISSUE TITLE
Import clojure.repl/doc

### ISSUE THREAD

**Issue by whamtet** [oxalorg/4ever-clojure#87]
Normal Clojure repls auto import `clojure.repl/doc` so that users can inspect doc strings from the command line.  It would be useful to have this in the 4ever-clojure repl as well.  I'm happy to open a PR for this issue if you agree it would be useful.

**Comment by borkdude** [comment]
Sounds good

**Comment by whamtet** [comment]
Thanks for the quick reply @borkdude , basically I need to create a new `doc` macro that wraps `clojure.repl/doc` like so

```clojure
(defmacro doc [sym]
  `(str "\n" (with-out-str (clojure.repl/doc ~sym))))
  ```
  
  Now you create the sci repl [here](https://github.com/oxalorg/4ever-clojure/blob/main/src/app/sci.cljs#L11-L14), so I need to supply my new `doc` to the repl.  However it is a macro, not a function.  How to supply a macro to the repl please?

**Comment by borkdude** [comment]
You don't need to wrap `clojure.repl/doc`, you can just evaluate `(require '[clojure.repl :refer [doc]])` before other evaluations.

**Comment by whamtet** [comment]
That gives

```
----- Error --------------------------------------
Message:  Attempting to call unbound fn: #'clojure.core/*print-fn*
Location: 1:1
```

So I need to wrap it in `with-out-str` or bind `clojure.core/*print-fn*` some other way.  In addition to that, it looks better with the extra newline at the beginning because of the layout of 4ever-clojure's repl.

**Comment by borkdude** [comment]
@whamtet I suggest doing:

```
(sci/alter-var-root sci/print-fn (constantly *print-fn*))
```

**Comment by whamtet** [comment]
hmm, if I run `(require '[sci.core :as sci])` it complains 'could not find namespace sci.core.'  Do I need to add extra config to `sci/init`?  Update version of sci?

**Comment by borkdude** [comment]
I mean to add this code outside of 4ever-clojure's REPL, near sci configuration of this project.

**Comment by borkdude** [comment]
Similar to this: https://github.com/babashka/nbb/blob/6734d02c577ce3d10967b5e75500ef7c3c44741a/src/nbb/core.cljs#L670

**Comment by whamtet** [comment]
That works!  But now its printing to the browser console, not the output panel.  That's why I wanted to wrap `clojure.repl/doc` with `with-out-str`.  Users who don't have dev tools open won't notice where its printing to.

**Comment by borkdude** [comment]
Ah yeah, make sense.

**Comment by whamtet** [comment]
Looks like there was one unnecessary line in the PR, I updated to remove that.

**Comment by oxalorg** [comment]
Thanks @whamtet! Fixed in #88
